packages/themeables/image/src/types.ts

```diff
@@ -1,5 +1,11 @@
 export type TThemeableImage = {
   resizeMode?: 'contain' | 'cover',
+  height: number,
```

Comment:
Should we make these ones optional? In case the desired behavior is that the image grows according to its own size.

Reply:
On, it!